ipynb/observationTable.md

```diff
@@ -0,0 +1,164 @@
+
+bioms = {
+    'gg'       : {'label': 'cr_gg_seqs',
+                  'biomsummary': path_root + '03-otus/01-closed-ref-greengenes/emp_cr_gg_13_8.summary.txt',
+                  'biomfile':    path_root + '03-otus/01-closed-ref-greengenes/emp_cr_gg_13_8.biom'},
+    'silva'    : {'label': 'cr_silva_seqs',
+                  'biomsummary': path_root + '03-otus/01-closed-ref-silva-16S/emp_cr_silva_16S.summary.txt',
+                  'biomfile':    path_root + '03-otus/01-closed-ref-silva-16S/emp_cr_silva_16S.biom'},                
+    'openref'  : {'label': 'or_gg_seqs',
+                  'biomsummary': path_root + '03-otus/02-open-ref-greengenes/emp_or.summary.txt',
+                  'biomfile':    path_root + '03-otus/02-open-ref-greengenes/emp_or.biom'},                  
+    'deblur100': {'label': 'deblur_100nt_seqs',
+                  'biomsummary': path_root + '03-otus/04-deblur/emp.100.min25.deblur.withtax.summary.txt',
+                  'biomfile':    path_root + '03-otus/04-deblur/emp.100.min25.deblur.withtax.biom'},                  
+    'deblur150': {'label': 'deblur_150nt_seqs',
+                  'biomsummary': path_root + '03-otus/04-deblur/emp.150.min25.deblur.withtax.summary.txt',
+                  'biomfile':    path_root + '03-otus/04-deblur/emp.150.min25.deblur.withtax.biom'},
+}
+fasta = {
+    'split-libraries' : {'label': 'split-libraries',
+                         'dir': path_root + '01-split-libraries',  
+                         'fnaFile': 'seqs.fna',          
+                         'logfile': 'split_library_log.txt'},
+    'adaptor-clean-up': {'label': 'adaptor-clean-up',
+                         'dir': path_root + '02-adaptor-clean-up', 
+                         'fnaFile': 'filtered_seqs.fna', 
+                         'filterFnaFile': 'seqs_to_filter.fna',
+                         'logfile': ''},
+}
+```
+
+
+```python
+# Gathering all necessary information from the original files themselves is not feasible, because of IO and CPU time.
+# The following method makes use of additional information that describe the BIOM tables and fasta files.
+# This is fine, as long as those descriptors are up to date. Otherwise, we are screwed!
+# These asumptions must hold for proper operations of this script:
+#     1. each BIOM file is acompanied by a *.summary.txt file which holds the number of counts per sample starting at line 15
+#     2. each seq.fasta file has its split_library_log.txt which give counts of sequences for the same sample. It can have several runs and counts might be split acros those runs.
+#     3. to avoid looking into the filered_seqs.fna files we substract the number of sequences to be filtered from the number of sequences per sample of the previous computation. Thus, it is neccessary that we have up-to-date 'seqs_to_filter.fna' files.
+
+############ STEP 1/4 ###########
+print("Step 1/4: parsing summary information about BIOM " + str(len(bioms)) + " tables: ", end="")
```

Comment:
Completely optional, but this may help clean up print statements in the future.

```python
print("Step 1/4: parsing summary information about BIOM  %d tables: " % len(bioms), end="")
```